src/adapter/src/catalog.rs

```diff
@@ -751,18 +751,14 @@ impl CatalogState {
     /// source.
     ///
     /// Note: Identifiers for the source, e.g. item name, are URL encoded.
-    pub fn try_get_webhook_url(
-        &self,
-        id: &GlobalId,
-        conn_id: Option<&ConnectionId>,
-    ) -> Option<url::Url> {
+    pub fn try_get_webhook_url(&self, id: &GlobalId) -> Option<url::Url> {
         let entry = self.try_get_entry(id)?;
-        let name = self.resolve_full_name(entry.name(), conn_id);
+        let name = self.resolve_full_name(entry.name(), entry.conn_id());
```

Comment:
This function returns a connection id if the entry is temporary, which webhook sources will never be. I think passing `None` here is correct.

Reply:
Good to know, updated!